test/test_helper.rb

```diff
@@ -10,4 +10,7 @@ class ActiveSupport::TestCase
   include ApplicationHelper
 
   # Add more helper methods to be used by all tests here...
+  def is_logged_in?
```

Comment:
__[rubocop]__
* Rename `is_logged_in?` to `logged_in?`. [:ref](https://github.com/bbatsov/ruby-style-guide#bool-methods-qmark)